bot/responses/not_sending_videos/transcription_handler_responses.py

```diff
@@ -6,14 +7,51 @@
 
 
 def get_no_quote_provided_message() -> str:
-    return "🔎 Podaj cytat, który chcesz znaleźć. Przykład: /klip Nie szkoda panu tego pięknego gabinetu?"
+    return "🔎 Podaj cytat, który chcesz znaleźć. Przykład: /transkrypcja Nie szkoda panu tego pięknego gabinetu?"
 
 
-def get_transcription_response(quote: str, context_segments: List[Dict[str, Union[int, str]]]) -> str:
-    response = f"🔍 Transkrypcja dla cytatu: \"{quote}\"\n\n```\n"
-    for segment in context_segments:
-        response += f"🆔 {segment["id"]} - {segment["text"]}\n"
+def get_transcription_response(quote: str, result: Dict[str, Union[str, List[Dict[str, str]]]]) -> str:
+    result_json = json.dumps(result)
+    result = json.loads(result_json)
```

Comment:
czemu po prostu nie korzystasz z dicta? xD json to w sumie `Dict[str, Any]`

Reply:
a bo ide gęgało XD to znalzłem sposób jak je uciszyć XD

Reply:
--------------------------------------------------
{'id': 197, 'start': 764.56, 'end': 767.56, 'text': ' Minerał jakiś?', 'author': '', 'comment': '', 'tags': ['', ''], 'location': '', 'actors': ['', ''], 'episode_info': {'season': 6, 'episode_number': 71, 'title': 'Dzieci rewolucji', 'premiere_date': '08.04.2012', 'viewership': 5973975}, 'video_path': 'bot/RANCZO-WIDEO/Sezon 6/Ranczo_S06E06.mp4'}
--------------------------------------------------
<class 'int'>
--------------------------------------------------
197
--------------------------------------------------
<class 'dict'>
--------------------------------------------------
<class 'dict'>
--------------------------------------------------


    print("-"*50)
    print(result['target'])
    print("-"*50)
    print(type(target['id']))
    print("-" * 50)
    print(target['id'])
    print("-" * 50)
    print(type(result['target']))
    print("-"*50)
    print(type(result))
    print("-"*50)
    
    czego to gęga ode mnie przy użyciu 'id' 
    
    Unexpected type(s): (str) Possible type(s): (SupportsIndex | slice) (SupportsIndex) (slice)